feat(protocol-designer): add to list of bugs addressed for PD 8.5 so far (#18169)
OpenTrons/opentrons-api: protocol-designer/release-notes.md

### Improved Features

- Protocol Designer includes a warning when you clear a slot with labware, a module, or a fixture used in a protocol step.

> - More robust validation when uploading custom labware.

Reviewer: maybe we could make this a bit more specific , or even just worded differently to _seem_ more specific, like: 

"Protocol Designer takes extra steps to validate any custom labware you upload."

and if we can think of an example, "PD takes extra steps, like....to validate..."

Author: Ah this is in relation to how AI was generating custom labwares that were wrong but still passed our previous validation logic. So to fix that, we added some additional testing to make sure the custom def is actually correct

### Bug Fixes

- Transfer step details show correct aspirate and dispense volumes when distributing liquid (a single aspirate and multiple dispenses).

> - While mixing, the pipette no longer lifts between mixes due to a newly created customizable push out field.

Author: i guess we should specify that by default, the push out is set to 0 now for a mix step but the field is also customizable if they want to change it to something else

Reviewer 2: I believe the default is pushout=0 for **all but the last dispense** in the mix?

Reviewer: Yep, I probably wrote this and it's confusing as written! Let's fix it.

During a mix, ``push_out=0`` by default to avoid multiple push out actions. You can choose your own push out volume in a mix step menu. 

*I'm assuming here that the push out is now customizable in the mix step menu. Not going to specify a max or min value since PD does that for you if you enter a value outside the acceptable range.

> - During a mix, `push_out=0` by default for all mixes except for the last, to avoid multiple push out actions. You can choose your own push out volume in a mix step menu.

Reviewer 3: No code formatting, since PD is a no-code tool.
```suggestion
- During a mix, push out is set to 0 by default for all mixes except for the last, to avoid multiple push out actions. You can choose your own push out volume in a mix step menu.
```

- When uploading a protocol and the drop tip location is unknown, users can now manually select the new location from the Drop Tip Location dropdown.
- Correctly migrate adapter/labware combo definitions into their singular definitions when uploading a protocol that is older than version 7.0.0.

> - When adding a disposal volume, a blow out location is now required.

Reviewer: Adding a few text suggestions to reword these: 

- Choose a new tip drop location from the dropdown menu when an uploaded protocol is missing a drop tip location. 

- Protocol Designer correctly updates adapter and labware combination definitions when you upload a protocol designed in Protocol Designer v7.0.0 or earlier. (could still use "migrate" here if we want)

> 

Reviewer 2: I'm trying to remember: was there some customer issue from March about us dropping or not dropping tips in some cases, that we fixed?

Author: I don't think so? this bullet point is referring to the most recent fix that Nick made to address the bug that you were investigating.

> - Protocol Designer correctly updates adapter and labware combination definitions when you upload a protocol designed in Protocol Designer v7.0.0 or earlier. (could still use "migrate" here if we want)

Reviewer 3: ```suggestion
- Protocol Designer correctly updates adapter and labware combination definitions when you upload a protocol designed in Protocol Designer v7.0.0 or earlier.
```
😇

Author: oops, classic example of me not ready the text and just blindly copy and pasting 😆

> - When adding a disposal volume, a blow out location is now required.

Reviewer 3: [noun](https://opentrons.atlassian.net/wiki/spaces/RPDO/pages/980746400/Opentrons+UX+Style+Guide#Verbs-vs.-nouns) 🧐 
```suggestion
- When adding a disposal volume, a blowout location is now required.
```
